src/handlers/graphical.rs

```diff
@@ -25,6 +25,7 @@ pub struct GraphicalReportHandler {
     pub(crate) theme: GraphicalTheme,
     pub(crate) footer: Option<String>,
     pub(crate) context_lines: usize,
+    pub(crate) tab_width: Option<usize>,
```

Comment:
This might sound dumb, but I wonder if a user should be allowed to use `tab_width = 0` for no space or it'd rather be better if `tab_width = 0` means using tabs 🤔